site2/docs/io-debezium-source.md

```diff
@@ -1,14 +1,15 @@
 ---
-id: io-debezium-source
-title: Debezium source onnector
+id: io-cdc-debezium
```

Comment:
why do you change this id? This is going to mess up all the documentation links. I would prefer not changing any doc id.

Reply:
I'll  fix it.

Reply:
I have fixed it,PTAL.